### DISCUSSION TITLE
Partidos políticos

### DISCUSSION THREAD

**Discussion by DafneCalvo** [PucelaBits/websegura#140] [Ideas]
¿Habéis pensado en la idea de añadir partidos políticos a la web? 

Es cierto que existe cierto riesgo de uso partidista de los datos. Pero también es cierto que los partidos políticos son esencialmente entidades orientadas al interés público. Son organizaciones creadas con el objetivo de gobernar y a hacer cambios en lo público, son quienes se responsabilizan de los ministerios, diputaciones, consellerias, etc., así como de sus webs. 

¿Qué pensáis?

**Comment by dariomehr** [comment]
Creo que es interesante, al igual que sería interesante el ibex35.
No sé cual fue el motivo por el que se decidió limitar el análisis únicamente a entidades públicas.
Pero si nos ceñimos a ese criterio, creo que los partidos políticos no están en ese conjunto.

**Reply by nukeador** [reply]
La decisión de limitarnos a las entidades públicas fue que son las únicas organizaciones a las que podemos pedir responsabilidades como ciudadanos porque sirven a los intereses públicos.

Una empresa privada sirve los intereses de sus accionistas, no tiene que dar cuentas a nadie más.

Dicho esto, creo que una vez perfeccionemos la plataforma para organismos públicos podemos abrir la pregunta de empresas privadas, pero creo que para eso cualquiera puede ir a observatory.mozilla.org y analizar la web que quiera. El listado de empresas privadas puede ser infinito.

**Reply by Cramer-0xbit** [reply]
estaria interesante añadirlas en un apartado aparte, aunque no sean estrictamente entidades públicas, los partidos son financiados por el estado mediante suvenciones no son organismos civiles idependientes de este y por lo tanto tendrían que cumplir de forma ejemplar la LOPD

**Reply by dariomehr** [reply]
> Una empresa privada sirve los intereses de sus accionistas, no tiene que dar cuentas a nadie más.

Pero los clientes y usuarios también pueden opinar. Muchas son tan grandes que tienen un impacto global, además de recibir contratos públicos. También algunas de las empresas privadas son monopolios-oligopolios y casi todos las usamos aunque no queramos. Deben respetar las regulaciones y leyes ciudadanas, así como preocuparse de su reputación con los clientes.

**Reply by nukeador** [reply]
Tienes razón @dariomehr 

Cuando lleguemos a ese punto habría que tener muy claro cual es el criterio y por qué unas sí y otras no. Quizá deberíamos tener un campo para introducir una web que no esté y que te mande al observatory para que te de el resultado.

**Comment by Cramer-0xbit** [comment]
aun así entiendo la preocupación de  @nukeador  de poder perder el norte si vamos difumunando los limites de lo que metemos y de lo que no

**Reply by Cramer-0xbit** [reply]
@DafneCalvo  sobre el interés partidista no creo que sea esencialmente negativo ya que los resultados  de la buena o mala implementación se van actualizando con cada cambio y estan dentro de su mano revertir  su "debilidad" en este argumento 
aparte cuando algo pasa a ser un interes partidista pasa a ser un interes dentro del debate político, metiendo aspectos de ciberseguridad dentro de los programas electorales y pasando a tener presente dichos aspectos en  la puesta a punto de nuevas políticas

**Comment by palmerabollo** [comment]
Cuando tengamos las 50 provincias con los organismos públicos, me parece buena idea pasar a otro tipo de organismos con fuerte vinculación o financiación pública (partidos políticos, fundaciones, asociaciones, ongs) y después considerar empresas privadas (tengo más dudas con esto ya que hay tantísimas que puede ser inabordable). Por ahora creo que es mejor poner foco en los sitios públicos.
